README.md

```diff
@@ -35,33 +35,94 @@ int(0)
 ### Parse an object of strings
 
 ```php
-$validationModel = new ObjectValidator(
+$personalInformationValidator = new ObjectValidator(
     (object) [
         'name' => new StringValidator(2, 30),
-        'city' => new StringValidator(2, 30)
+        'city' => new StringValidator(2, 30),
+        'age' => new IntegerValidator(1, 200),
     ],
-    ['name', 'city'], //required properties
+    ['name', 'city', 'age'], //required properties
     false //no additional properties allowed
 );
 
-$value = $validationModel->parse((object) [
-    'name' => 'Xenofon',
-    'city' => 'Thessaloniki'
+$personalInformation = $validationModel->parse((object) [
+    'name' => 'Jane Doe',
+    'city' => 'Athens',
+    'age' => 28
 ]);
 
-print_r($value);
+print_r($personalInformation);
 ```
 
 The above example will output:
 
 ```
 stdClass Object
 (
-    [name] => Xenofon
-    [city] => Thessaloniki
+    [name] => Jane Doe
+    [city] => Athens,
+    [age] => 28
 )
 ```
 
+### Validating an array of enum strings
+
+```
+    /*
+     * A validator that allows you to pick one or two colors between blue, green and red
+     */
+    $colorsValidator = new ArrayValidator(
+        $minItems = 1,
```

Comment:
why the `variable = something`? instead of adding comments?

Reply:
hmm, you right comment could work to